ojbackend/install.py

```diff
@@ -0,0 +1,40 @@
+"""
+	install.py
+	This file let users to set up machineInfo.config and ojdatabase.config file
+"""
+
+from bash import bash
+def set_MachineInfo():
+    
+    bash('test -d machineInfo.config  && rm -r machineInfo.config')    
```

Comment:
Would it be better if I use Popen or check_call?... I didn't consider too much then.